### ISSUE TITLE
ReferenceError: __spreadProps is not defined starting v. 3.9.0

### ISSUE THREAD

**Issue by ctrl-brk** [mapbox/mapbox-gl-js#13376]
Starting v 3.9 I'm getting `__spreadProps is not defined` error in my Angular app.

**mapbox-gl-js version**: 3.9.x

**browser**:  Chrome, FF

### Link to Demonstration
Non-working 3.9:
https://stackblitz.com/edit/stackblitz-starters-cga9vcev?file=src%2Fmain.ts
Working 3.8:
https://stackblitz.com/edit/stackblitz-starters-mfty1jc8?file=src%2Fmain.ts

**Comment by shuabrannigan** [comment]
I am having same issue in my recently updated Angular v19 app in v3.9.1, 
i rolled back to v3.8 which fixed the __spreadProps issue but have a new issue where the map on first load does not fill the container correctly.

solution currently is to force map.resize which isn't ideal.

**Comment by ctrl-brk** [comment]
> I am having same issue in my recently updated Angular v19 app in v3.9.1, i rolled back to v3.8 which fixed the __spreadProps issue but have a new issue where the map on first load does not fill the container correctly.
> 
> solution currently is to force map.resize which isn't ideal.

Yep, that's too. One way around is to set container size directly in style attr.
ex: <div style='height: 100vh; width: 100vw'>

If you have it in .css it's not working anymore.

**Comment by shuabrannigan** [comment]
@ctrl-brk thanks for that, solved the issue i was having without needing to force a resize event.

**Comment by daviidmart** [comment]
Same here, version 3.9.1 doesn't work with Angular 19.0.5 and TypeScript 5.6.3.

**Comment by mourner** [comment]
Thanks for the report — on it, will try to include a fix for this in v3.9.2.

**Comment by EthanTuning** [comment]
Yes, this is plaguing the progress of a new app I am working on right now. 

Not sure any additional information I could provide, but yes reverting back to `3.8.0` does fix the issue.
